src/commands/init.js

```diff
@@ -19,33 +23,85 @@ function handleInitError(rawError) {
       return ERROR_MESSAGE_NOT_ADMIN_USER;
     case 'Not Found':
       return ERROR_MESSAGE_PROJECT_NOT_FOUND;
+    case 'No production/remote environment.':
+      return ERROR_MESSAGE_NO_PRODUCTION_OR_REMOTE_ENVIRONMENT;
+    case 'A user can have only one development environment per project.':
+      return ERROR_MESSAGE_ENVIRONMENT_OWNER_UNICITY;
+    case 'An environment with this name already exists. Please choose another name.':
+      return ERROR_MESSAGE_ENVIRONMENT_OWNER_UNICITY;
     default:
       return error;
   }
 }
 
+function validateEndpoint(input) {
+  if (!/^https?:\/\/.*/i.test(input)) {
+    return 'Application input must be a valid url.';
+  }
+  if (!/^http((s:\/\/.*)|(s?:\/\/(localhost|127\.0\.0\.1).*))/i.test(input)) {
+    return 'HTTPS protocol is mandatory, except for localhost and 127.0.0.1.';
+  }
```

Comment:
[Code Style] Those two regular expressions could be stored into constants at the start of the file